src/main/java/com/github/reducktion/socrates/validator/IdValidator.java

```diff
@@ -18,6 +18,7 @@ static IdValidator newInstance(final Country country) {
             case BE: return new BelgiumIdValidator();
             case BR: return new BrazilIdValidator();
             case CA: return new CanadaIdValidator();
+            case DE: return new GermanyIdValidator();
```

Comment:
Can you add a test for this new validator, pls?
We usually do this in the class `IdValidatorTest`.

Reply:
I'm sorry. I almost missed this. 😬

Reply:
Sure, will do.
Good catch.

Reply:
Done.